### PR TITLE
fix indentation

### LINKED ISSUE #133: Fix indentation in example requests/responses
Many of the examples for HTTP requests/responses are indented by two spaces by default, which is not required, and some indentations are inconsistent. The extra indentations should be removed and the examples should be checked for consistency.

### PR DESCRIPTION
**Pull request by Sakurann**
resolves #133

### CODE REVIEW

File: examples/response/jarm_jwt_vc_json_post.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-  POST /post HTTP/1.1
-    Host: client.example.org
-    Content-Type: application/x-www-form-urlencoded
+POST /post HTTP/1.1
+  Host: client.example.org
+  Content-Type: application/x-www-form-urlencoded
```
Comment (c2bo) [inline examples/response/jarm_jwt_vc_json_post.txt:3]: Shouldn't the headers be on the same level?
```suggestion
POST /post HTTP/1.1
Host: client.example.org
Content-Type: application/x-www-form-urlencoded
```

**Review by c2bo — CHANGES_REQUESTED** [review]

File: openid-4-verifiable-presentations-1_0.md [hunk 2]
```diff
@@ -692,13 +692,13 @@ The value of the redirect URI is an absolute URI as defined by [@!RFC3986] Secti
...
-  Cache-Control: no-store
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
 
-  {
-    "redirect_uri":"https://client.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee" 
-  }
+{
+  "redirect_uri":"https://client.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee"
```
Comment (c2bo) [inline openid-4-verifiable-presentations-1_0.md:700]:
```suggestion
  "redirect_uri": "https://client.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee" 
```

File: openid-4-verifiable-presentations-1_0.md [hunk 3]
```diff
@@ -642,40 +642,40 @@ The following is a non-normative example of the payload of a Request Object with
    "response_mode": "direct_post",
    "presentation_definition": {...},
    "nonce": "n-0S6_WzA2Mj",
-   "state" : "eyJhb...6-sVA
+   "state" : "eyJhb...6-sVA"
```
Comment (c2bo) [inline openid-4-verifiable-presentations-1_0.md:645]:
```suggestion
   "state": "eyJhb...6-sVA"
```

Looking at this and then at the draft I realized that the JSON examples are also somewhat inconsistent. Some examples seem to be using 3 space indentation (like here), others 2 or 4. Should i create another Issue / PR for this?

**Review by c2bo — APPROVED** [review]
As commented, I do believe we need another pass at quite a few of the examples to get consistent indentation for everything, but the http request/response examples seem to be fine now.

**Comment by Sakurann** [comment]
Which examples..?

**Comment by c2bo** [comment]
> Which examples..?

https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-5.7-8 For example - header and body use different indentation. There were a few examples were e.g., the JSON formatting seemed to be different. I can take a bit of time and collect everything I find into another issue/PR.

**Comment by Sakurann** [comment]
> I can take a bit of time and collect everything I find into another issue/PR.

@c2bo thank you - that would be great!

**Pull request merged**